pkgs/development/python-modules/pytest-flake8/default.nix

```diff
@@ -3,7 +3,7 @@
 buildPythonPackage rec {
   name = "${pname}-${version}";
   pname = "pytest-flake8";
-  version = "0.9.1";
+  version = "1.0.0";
 
   # although pytest is a runtime dependency, do not add it as
   # propagatedBuildInputs in order to allow packages depend on another version
```

Comment:
I believe this comment should be removed and `pytest` moved to `propagatedBuildInputs`, because applications can use their own `pytest` version using `packageOverrides`. Shouldn't it, @FRidh?

Reply:
In case of `pytest` it is not just for applications, but also for libraries. Certain libraries may request a certain `pytest` version.

Reply:
So I'll keep it as it is or change it somehow?

Reply:
It can stay as it is.

Reply:
Ok, thanks! Then I think this should be ready to merge.